packages/opencensus-exporter-zpages/src/zpages-frontend/page-handlers/tracez.page-handler.ts

```diff
@@ -91,27 +100,22 @@ const getCanonicalCode = (status: number) => {
  * This change should be made in the core module.
  * @param inputSpan The span to serialize.
  */
-function serializeSpan(inputSpan: Span|RootSpan): Partial<Span> {
-  const span: Partial<Span> = {
+function serializeSpan(inputSpan: Span): SerializedSpan {
+  const span: SerializedSpan = {
     startTime: inputSpan.startTime,
     duration: inputSpan.duration,
     traceId: inputSpan.traceId,
     id: inputSpan.id,
-    parentSpanId: inputSpan.id
+    parentSpanId: inputSpan.id,
+    spans: undefined
   };
-  if (inputSpan.isRootSpan) {
-    // We possibly need to assign the spans field that is only available
-    // on root spans. The core module doesn't export this as an
-    // exportable field.
-    // tslint:disable-next-line:no-any
-    (span as any).spans =
-        (inputSpan as RootSpan).spans.map((childSpan: Span) => ({
-                                            startTime: childSpan.startTime,
-                                            id: childSpan.id,
-                                            name: childSpan.name,
-                                            duration: childSpan.duration
-                                          }));
-  }
+  // tslint:disable-next-line:no-any
```

Comment:
Since this doesn't use `any` below, can you remove this tslint disable comment?

Reply:
Removed